Reference/conformance/conformance.pb.swift

```diff
@@ -230,11 +230,28 @@ struct Conformance_ConformanceResponse: SwiftProtobuf.Message {
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   enum OneOf_Result: Equatable {
```

Comment:
I wonder if we should put some canned comment on this to better tie it to the oneof field and/or the oneof comment?

> Used for the oneof field [name].

or

> Used for the oneof field [name].
>
> [oneof field comment]